src/reference/00-Getting-Started/07A-Task-Graph.md

```diff
@@ -0,0 +1,324 @@
+---
+out: Task-Graph.html
+---
+
+  [Basic-Def]: Basic-Def.html
+  [Scopes]: Scopes.html
+  [Make]: https://en.wikipedia.org/wiki/Make_(software)
+  [Ant]: http://ant.apache.org/
+  [Rake]: https://ruby.github.io/rake/
+
```

Comment:
This section should start off defining the key terms:

- Task
- Setting

With one/two-liner descriptions. The details of setting dependencies, syntax, etc. can come later, but a paragraph that makes it clear "there are tasks and settings and they are similar but different things" would I think help greatly for anyone trying to understand subsequent exposition